tests/fixtures/long.md

```diff
@@ -17,16 +17,33 @@ function foo() {
 }
 ```
 
+<!-- eslint-env node -->
+<!-- eslint-disable eol-last, quotes -->
+
 ```node
 console.log(process.version);
 ```
 
 How about some JSX?
 
+<!--
+    eslint quotes: [
+        "error",
+        "single"
+    ]
+-->
+<!--eslint-disable no-console-->
+
 ```jsx
 console.log("Error!");
 ```
 
     I may be a code block, but don't lint me!
 
+<!-- eslint-disable -->
```

Comment:
Since ESLint only supports those as line comments, those wouldn't have any effect inserted at the top of the code block. There's nothing preventing using them inside though. Is there a need to be able to disable individual lines from outside the code block?